.azure/pipelines/fast-pr-validation.yml

```diff
@@ -18,12 +18,61 @@ phases:
 - phase: RepoBuilds
   queue:
     name: Hosted VS2017
-    parallel: 2
+    parallel: 3
     matrix:
       DataProtection:
         _FolderName: DataProtection
       WebSockets:
         _FolderName: WebSockets
+
+- template: jobs/iisintegration_job.yml
+  parameters:
+    jobName: IISIntegration_IIS_Windows
+    buildArgs: "/p:SkipIISBackwardsCompatibilityTests=true /p:SkipIISExpressTests=true /p:SkipIISForwardsCompatibilityTests=true"
```

Comment:
Looks like there is a lot of redundancy. Another way to do this is with a `matrix`. https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=vsts&tabs=schema#matrix

Something like this ... (didn't fully flesh out all combinations, but you get the idea):

```yml

- template: jobs/iisintegration_job.yml
  parameters:
    variables:
      _FolderName: IISIntegration
    matrix:
      IISIntegration_IIS:
        TestGroupName: IIS
        SkipIISBackwardsCompatibilityTests: true
        SkipIISExpressTests: true
        SkipIISForwardsCompatibilityTests: true
      IISIntegration_IISExpress:
        TestGroupName: IISExpress
        SkipIISBackwardsCompatibilityTests: true
        SkipIISTests: true
        SkipIISForwardsCompatibilityTests: true
      IISIntegration_IISForwardCompat:
        TestGroupName: IISForwardCompat
        SkipIISBackwardsCompatibilityTests: true
        SkipIISTests: true
        SkipIISForwardsCompatibilityTests: false
    jobName: IISIntegration_$(TestGroupName)
    buildArgs: "/p:SkipIISBackwardsCompatibilityTests=$(SkipIISBackwardsCompatibilityTests) /p:SkipTests=$(SkipIISTests)  /p:SkipIISExpressTests=$(SkipIISExpressTests) /p:SkipIISForwardsCompatibilityTests=$(SkipIISForwardsCompatibilityTests)"

```

Reply:
Yeah this is much better.